types/yosensi/YO-PurePro/co.schema.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$id": "https://akenza.io/yosensi/YO-PurePro/co.schema.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "processingType": "uplink_decoder",
+    "topic": "co",
+    "title": "CO",
+    "properties": {
+      "co": {
+        "title": "CO Concentration",
+        "type": "number",
+        "description": "CO Concentration in ppm.",
+        "unit": "ppm",
+        "minimum": 0,
+        "maximum": 1000,
+        "hideFromKpis": true
```

Comment:
setting hideFromKpis to true will disable displaying the value on the device details page in the akenza platform, we therefore recommend to only set this to true if you would like to hide the value (e.g. internal or debugging information)

Reply:
Alright, thanks for clarification, I will do the changes and push them together with the formatted documents. (using Prettier)